snorkeling_helper/candidates_helper/encode_candidates_modules.py

```diff
@@ -0,0 +1,57 @@
+import tqdm
+
+
+def encode_lemmas(
+    candidate_df,
+    stopwords,
+    word_mapper,
+    entity_fieldnames=["disease_start", "disease_end", "gene_start", "gene_end"],
+    entity_one="DISEASE_ENTITY",
+    entity_two="GENE_ENTITY",
+):
```

Comment:
Perhaps adding a description here would be helpful. I forget what these lemmas are

Reply:
+1, except that I know what a lemma is but not why you would encode one

Reply:
>Perhaps adding a description here would be helpful. I forget what these lemmas are

Lemmas are the root form of words i.e. The word "are" -> "be" or "walked" -> "walk". In NLP taking the root form of worlds helps lower the dimensionality of my training set when I get ready to train the network as well as keep sentences simple.

>+1, except that I know what a lemma is but not why you would encode one

Ah this step is for the transformer. In order to get the network to understand my sentences I needed to have an encoding layer to get a training matrix of (batch size, sequence size, embedding size). Right now it doesn't make much sense but one-two PRs from now it will click.